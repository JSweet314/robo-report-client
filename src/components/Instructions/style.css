.instructions {
  display: flex;
  flex-direction: column;
  justify-content: center;
  align-items: center;
  width: 50%;
  font-size: 2rem;
  background-color: white;
  padding: 2rem;
  border-radius: 15px;
  box-shadow: 2px 2px 6px 2px black;
}

.instructions__heading {
  text-align: left;
  width: 100%;
}

.submit-new-complaint-btn {
  font-weight: bold;
  display: block;
  width: fit-content;
  margin: 1em auto;
  padding: .2em;
  border: 1px solid darkgray;
  border-radius: 10px;
  box-shadow: 2px 2px 2px gray;
  padding: 1rem;
}

.submit-new-complaint-btn:hover {
  color: white;
  background-color: #467EB4;
}