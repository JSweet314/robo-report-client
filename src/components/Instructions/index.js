import React from 'react';
import './style.css';

const Instructions = () => {
  return (
    <div className="instructions">
      <h4 className='instructions__heading'>How it works:</h4>
      <p className='instructions__body'>
        When you choose to submit a new complaint, you will be taken through
        all of the questions needed to complete the report. The questions are
        taken directly from the FCC&#39;s complaint form for unwanted calls.
        Once you have completed all of the required parts of the form, Robo
        Report will pull up the FCC form and complete it for you! To finish
        the process, all you will need to do is:
        <ol>
          <li>
            Scoll through and review the form and make any edits you&#39;d
            like.
          </li>
          <li>Check the Captcha that you are not a robot (ironic, we know)</li>
          <li>Submit the form!</li>
        </ol>
      </p>
    </div>
  );
};

export default Instructions;