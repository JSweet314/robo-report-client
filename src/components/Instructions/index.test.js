import React from 'react';
import { shallow } from 'enzyme';
import Instructions from "./index";

describe('Instructions', () => {
  let wrapper;
  beforeEach(() => {
    wrapper = shallow(<Instructions />);
  });

  it('should match a snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });
});