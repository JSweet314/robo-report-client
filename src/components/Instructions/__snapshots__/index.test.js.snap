// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Instructions should match a snapshot 1`] = `
<div
  className="instructions"
>
  <h4
    className="instructions__heading"
  >
    How it works:
  </h4>
  <p
    className="instructions__body"
  >
    When you choose to submit a new complaint, you will be taken through all of the questions needed to complete the report. The questions are taken directly from the FCC's complaint form for unwanted calls. Once you have completed all of the required parts of the form, Robo Report will pull up the FCC form and complete it for you! 
    <br />
    <br />
    To finish the process, all you will need to do is:
  </p>
  <ol>
    <li>
      Scoll through and review the form and make any edits you'd like.
    </li>
    <li>
      Check the Captcha that you are not a robot (ironic, we know)
    </li>
    <li>
      Submit the form!
    </li>
  </ol>
  <Link
    className="submit-new-complaint-btn"
    replace={false}
    to="/newComplaint"
  >
    Submit New Complaint
  </Link>
</div>
`;
